src/main/java/org/example/Extractor.java

```diff
@@ -13,13 +14,45 @@ public List<Chair> getChairs(String url){
 
         String role, name, email;
         List<Chair> chairs = new ArrayList<>();
-        //todo
+        Document chairPage;
+        try{
+            chairPage = Jsoup.connect(url).get();
```

Comment:
connect.get() is common in multiple methods. Can we make a separate method that returns a page, and everyone calls it?